daemon/src/main/java/org/sonarlint/daemon/services/ConnectedSonarLintImpl.java

```diff
@@ -51,9 +51,14 @@
 import org.sonarsource.sonarlint.daemon.proto.SonarlintDaemon.Void;
 
 public class ConnectedSonarLintImpl extends ConnectedSonarLintGrpc.ConnectedSonarLintImplBase {
-  private static final Logger LOGGER = LoggerFactory.getLogger(ConnectedSonarLintImpl.class);
   private ConnectedSonarLintEngine engine;
-  private ProxyLogOutput logOutput = new ProxyLogOutput();
+  private final ProxyLogOutput logOutput;
+  private final Logger logger;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312)